bundles/org.openhab.binding.tesla/README.md

```diff
@@ -339,6 +348,13 @@ sitemap main label="Main"
         {
             Mapview item=TeslaLocation height=10
         }
+        Frame
+        {
+            String item=TeslaDestinationName
+            Number item=TeslaMinutesToArrival
+            Number item=TeslaDistanceToArrival
```

Comment:
Have you tested these examples? I'm not familiar with `String` and `Number` for sitemaps, and they don't seem documented: https://www.openhab.org/docs/ui/sitemaps.html#element-types. Most likely `Default` or `Text` should be used.

Reply:
Yes, I stumbled upon this too while creating my own mobile sitemap. Will correct the readme.